reV/SAM/generation.py

```diff
@@ -624,6 +625,256 @@ def default(self):
         return self._default
 
 
+class SolarThermal(Solar):
+    """ Base class for solar thermal """
+    def __init__(self, resource=None, meta=None, parameters=None,
+                 output_request=None):
+        """Initialize a SAM solar water heating object.
+
+        """
+        self.timezone = None
```

Comment:
Don't use timezone as an attribute. Not necessary, only used once, and inconsistent with the other gen objects.

Reply:
Removed.